src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -268,6 +281,17 @@ public async Task<int> GenerateAIReview(string reviewId, string revisionId)
             return result.Violations.Count;
         }
 
+        private async Task ToggleReviewApproval(ClaimsPrincipal user, ReviewListItemModel review, string notes)
+        {
+            await ManagerHelpers.AssertApprover<ReviewListItemModel>(user, review, _authorizationService);
+            var userId = user.GetGitHubLogin();
+            var changeUpdate = ChangeHistoryHelpers.UpdateBinaryChangeAction<ReviewChangeHistoryModel, ReviewChangeAction>(
+                review.ChangeHistory, ReviewChangeAction.Approved, userId, notes);
+            review.ChangeHistory = changeUpdate.ChangeHistory;
+            review.IsApproved = changeUpdate.ChangeStatus;
```

Comment:
Is this approval for first release?

Reply:
Yes, Review approval is approval for first release.